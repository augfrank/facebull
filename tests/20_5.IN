M0   C0   C2  2275
M1   C0   C3  2579
M2   C0   C6  2608
M3   C0   C7  2715
M4   C0   C8  2277
M5   C0   C9  2163
M6   C0   C10  2891
M7   C0   C13  2977
M8   C0   C14  2076
M9   C0   C19  2506
M10   C1   C2  2851
M11   C1   C3  2606
M12   C1   C5  2145
M13   C1   C6  2803
M14   C1   C8  2585
M15   C1   C9  2448
M16   C1   C10  2845
M17   C1   C11  2563
M18   C1   C12  2645
M19   C1   C16  2484
M20   C1   C19  2907
M21   C2   C0  2732
M22   C2   C1  2746
M23   C2   C3  2577
M24   C2   C5  2606
M25   C2   C10  2572
M26   C2   C11  2640
M27   C2   C12  2153
M28   C2   C13  2980
M29   C2   C14  2655
M30   C2   C15  2755
M31   C2   C19  2534
M32   C3   C1  2056
M33   C3   C2  2072
M34   C3   C4  2114
M35   C3   C5  2212
M36   C3   C6  2637
M37   C3   C7  2544
M38   C3   C9  2457
M39   C3   C10  2487
M40   C3   C13  2311
M41   C3   C14  2946
M42   C3   C17  2066
M43   C4   C0  2870
M44   C4   C5  2718
M45   C4   C6  2977
M46   C4   C7  2833
M47   C4   C9  2157
M48   C4   C11  2925
M49   C4   C12  2169
M50   C4   C13  2475
M51   C4   C16  2103
M52   C4   C17  2825
M53   C4   C19  2625
M54   C5   C1  2897
M55   C5   C2  2930
M56   C5   C3  2313
M57   C5   C6  2871
M58   C5   C8  2334
M59   C5   C9  2453
M60   C5   C11  2974
M61   C5   C13  2774
M62   C5   C15  2249
M63   C5   C18  2931
M64   C5   C19  2323
M65   C6   C1  2883
M66   C6   C2  2271
M67   C6   C7  2682
M68   C6   C8  2360
M69   C6   C9  2294
M70   C6   C11  2805
M71   C6   C12  2592
M72   C6   C13  2720
M73   C6   C14  2165
M74   C6   C15  2086
M75   C6   C17  2121
M76   C6   C18  2518
M77   C6   C19  2702
M78   C7   C1  2940
M79   C7   C3  2076
M80   C7   C4  2776
M81   C7   C8  2564
M82   C7   C9  2444
M83   C7   C11  2855
M84   C7   C12  2453
M85   C7   C13  2885
M86   C7   C16  2098
M87   C7   C17  2999
M88   C7   C19  2376
M89   C8   C1  2194
M90   C8   C2  2845
M91   C8   C3  2200
M92   C8   C4  2841
M93   C8   C6  2180
M94   C8   C7  2788
M95   C8   C10  2709
M96   C8   C11  2648
M97   C8   C12  2219
M98   C8   C13  2923
M99   C8   C15  2749
M100   C8   C16  2344
M101   C8   C17  2730
M102   C8   C19  2301
M103   C9   C0  2471
M104   C9   C1  2597
M105   C9   C2  2183
M106   C9   C3  2058
M107   C9   C4  2137
M108   C9   C10  2447
M109   C9   C13  2999
M110   C9   C14  2439
M111   C9   C16  2868
M112   C9   C18  2672
M113   C10   C0  2968
M114   C10   C1  2431
M115   C10   C4  2636
M116   C10   C5  2206
M117   C10   C7  2792
M118   C10   C8  2123
M119   C10   C13  2569
M120   C10   C14  2239
M121   C10   C16  2859
M122   C10   C18  2450
M123   C11   C0  2667
M124   C11   C1  2973
M125   C11   C2  2508
M126   C11   C3  2068
M127   C11   C4  2316
M128   C11   C5  2705
M129   C11   C6  2193
M130   C11   C9  2318
M131   C11   C10  2976
M132   C11   C12  2835
M133   C11   C13  2463
M134   C11   C18  2889
M135   C12   C1  2986
M136   C12   C3  2102
M137   C12   C4  2386
M138   C12   C5  2341
M139   C12   C8  2499
M140   C12   C10  2921
M141   C12   C11  2207
M142   C12   C13  2822
M143   C12   C14  2908
M144   C12   C15  2247
M145   C12   C17  2653
M146   C12   C18  2826
M147   C12   C19  2953
M148   C13   C1  2619
M149   C13   C2  2089
M150   C13   C3  2588
M151   C13   C7  2991
M152   C13   C8  2017
M153   C13   C9  2612
M154   C13   C10  2851
M155   C13   C17  2185
M156   C13   C18  2286
M157   C13   C19  2322
M158   C14   C2  2107
M159   C14   C4  2861
M160   C14   C5  2432
M161   C14   C6  2920
M162   C14   C8  2994
M163   C14   C9  2280
M164   C14   C12  2704
M165   C14   C13  2618
M166   C14   C15  2996
M167   C14   C17  2759
M168   C14   C19  2821
M169   C15   C0  2633
M170   C15   C2  2288
M171   C15   C5  2179
M172   C15   C11  2129
M173   C15   C12  2514
M174   C15   C14  2329
M175   C15   C16  2437
M176   C15   C17  2876
M177   C15   C18  2324
M178   C15   C19  2810
M179   C16   C1  2027
M180   C16   C7  2916
M181   C16   C8  2437
M182   C16   C9  2488
M183   C16   C12  2554
M184   C16   C14  2210
M185   C16   C17  2975
M186   C16   C18  2294
M187   C17   C1  2779
M188   C17   C2  2538
M189   C17   C3  2976
M190   C17   C4  2746
M191   C17   C9  2809
M192   C17   C12  2899
M193   C17   C14  2174
M194   C17   C15  2205
M195   C17   C16  2775
M196   C17   C19  2251
M197   C18   C3  2574
M198   C18   C6  2149
M199   C18   C7  2287
M200   C18   C9  2122
M201   C18   C13  2727
M202   C18   C14  2513
M203   C18   C15  2524
M204   C18   C16  2791
M205   C18   C19  2645
M206   C19   C0  2698
M207   C19   C2  2308
M208   C19   C3  2477
M209   C19   C4  2217
M210   C19   C5  2075
M211   C19   C6  2696
M212   C19   C7  2758
M213   C19   C8  2903
M214   C19   C10  2948
M215   C19   C12  2152
M216   C19   C13  2008
M217   C19   C14  2691
M218   C19   C15  2067
M219   C19   C16  2316
