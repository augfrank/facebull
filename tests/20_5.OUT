43727
8 12 27 34 47 62 75 86 93 108 118 123 136 156 158 172 179 196 199 216
